### PR TITLE
📝  New Collection CSS Frameworks

### PR DESCRIPTION
**Pull request by krishdevdb**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> A CSS framework is a set of tools that help you build websites. And People Spend Long Time Finding The Perfect One For Them

---------------------------------------------------------------------

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: collections/css-frameworks/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+ - jgthms/bulma
+ - uikit/uikit
+ - semantic-org/semantic-ui
+ - Dogfalo/materialize
+ - pure-css/pure
+ - tailwindlabs/tailwindcss
+display_name: CSS Frame Works
+created_by: krishdevdb
+---
+A CSS framework is a set of css classes that allow you to create your website with few or no css code.
```
Comment (gallexi) [inline collections/css-frameworks/index.md:14]:
```suggestion
A CSS framework is a set of css classes that allow you to create your website with little to no new css code.
```

**Review by gallexi — APPROVED** [review]
Small suggestion to make the English a bit more idiomatic, but otherwise this looks good! Thanks!

**Comment by krishdevdb** [comment]
Hi @gallexi I amde the changes you were asking for. Please review the changes and let me know for some issues or modifications

**Review by smashwilson — APPROVED** [review]
Perfect. Thanks for contributing! 🙌🏻

**Comment by krishdevdb** [comment]
> Perfect. Thanks for contributing! 🙌🏻

Thanks For Accepting My Contribution @smashwilson and @gallexi

**Comment by jess6969** [comment]
Okay thank you 😊

**Pull request merged**